### PR TITLE
Feat/add fee btc

### PR DESCRIPTION
**Pull request by toandq2009**
- add show fee btc on extension
- update logic for show history bitcoin
- Add bitcoin faucet for bitcoin testnet
- remove all console.log

### CODE REVIEW

**Review by marxeille — DISMISSED** [review]

File: packages/extension/src/pages/send-btc/index.tsx [hunk 1]
```diff
@@ -155,12 +156,20 @@ export const SendBtcPage: FunctionComponent<{
...
                     notification.push({
                       placement: 'top-center',
                       type: tx ? 'success' : 'danger',
                       duration: 5,
-                      content: tx ? `Transaction successful with tx: ${tx}` : `Transaction failed with tx: ${tx}`,
+                      content: tx ? (
+                        <div className="alert-inner--text">
+                          Transaction successful with tx:{' '}
+                          <a target="_blank" href={`https://live.blockcypher.com/btc-testnet/tx/${tx}/`}>
+                            {Address.shortAddress(tx)}
```
Comment (haunv3) [inline packages/extension/src/pages/send-btc/index.tsx:169]: Should we create a constant for this link?

File: packages/bitcoin/src/__tests__/helper.spec.js [hunk 2]
```diff
@@ -210,17 +208,15 @@ describe('helper', () => {
       {
         selectedCrypto: 'bitcoin',
         keyDerivationPath: '84',
-        mnemonic:
-          'wrist illness circle evidence accident loan thing mystery output inhale fat rookie'
+        mnemonic: 'wrist illness circle evidence accident loan thing mystery output inhale fat rookie'
```
Comment (haunv3) [inline packages/bitcoin/src/__tests__/helper.spec.js:212]: Do we need to describe further this is a key test?

File: packages/extension/src/pages/main/account.tsx [hunk 3]
```diff
@@ -167,6 +167,13 @@ export const AccountView: FunctionComponent = observer(() => {
           (selected?.bip44HDPath?.addressIndex ?? '0')
         }`}
       </div>
+      {chainId === ChainIdEnum.BitcoinTestnet && (
+        <div className={styleAccount.coinType}>
+          <a target="_blank" href="https://bitcoinfaucet.uo1.net/">
```
Comment (haunv3) [inline packages/extension/src/pages/main/account.tsx:172]: Do we need to add constants to it? I see it used in a number of places

File: packages/mobile/src/modals/permission/index.tsx [hunk 4]
```diff
@@ -25,7 +25,6 @@ export const AccessModal: FunctionComponent<{
       if (waitingData) {
         await permissionStore.reject(waitingData.id);
         if (permissionStore.waitingBasicAccessPermissions.length === 0) {
```
Comment (haunv3) [inline packages/mobile/src/modals/permission/index.tsx]: Please delete it if we don't do anything here

File: packages/extension/src/pages/main/account.tsx (hunk 3 above)
Comment (toandq2009) [reply packages/extension/src/pages/main/account.tsx:172]: Now, It only show link on extension, mobile show on webview page

**Review by haunv3 — APPROVED** [review]

**Review by marxeille — APPROVED** [review]

**Pull request merged**